docs/concepts/data-views.asciidoc

```diff
@@ -143,10 +143,11 @@ all clusters having a name starting with `cluster_`:
 `cluster_*:logstash-*,cluster_*:-logstash-old*`
 ```
 
-To exclude a cluster having a name starting with `cluster_`:
+To exclude a cluster having a name starting with `cluster_`. Excluding a cluster avoids sending any network calls to that cluster. For more information, see
```

Comment:
I know you copied the first sentence from the original, but I think the original was incorrect. Shouldn't it say:
> To exclude a cluster having a name starting with `cluster_one`.

?